packages/bitbucket-code-insights/src/index.ts

```diff
@@ -0,0 +1,24 @@
+import fetch from 'node-fetch';
+import { HttpProxyAgent } from 'http-proxy-agent';
+import { BitbucketReportBody } from "./types"
+
+async function uploadReportToBitbucket(externalId: string, body: BitbucketReportBody){
+
+  const url = `http://api.bitbucket.org/2.0/repositories/${process.env.BITBUCKET_REPO_FULL_NAME}/commit/${process.env.BITBUCKET_COMMIT}/reports/${externalId}`;
+
+  console.log('url', url);
+
+  const response = await fetch(url, {
+    agent: new HttpProxyAgent('http://localhost:29418'),
```

Comment:
What is this port? Can we have a cost for it? Is something we want to hard code?